README.md

```diff
@@ -10,6 +10,87 @@ minutes, not days.
 $ npx create-choo-app <project-directory>
 ```
 
+## ⚠️  HTTPS Instructions
+When you first open up your application in a browser, you'll probably see a
+warning page about HTTPS connections being untrusted. No worries, this is
+entirely expected behavior. Follow the instructions below to solve this for
+your browser.
+
+<details>
+  <summary>
+    <b>How does this work?</b>
+  </summary>
+  For HTTPS to run on <code>localhost</code>, we must sign a TLS certificate
+  locally. This is better known as a "self-signed certificate". Browsers
+  actively check for certificates from uknown providers, and warn you (for good
+  reason!) In our case, however, it's safe to ignore.
+
+  HTTPS is needed for an increasing amount of APIs to work in the browser. For
+  example if you want to test HTTP/2 connections or use parts of the storage
+  API, you have no choice but to use an HTTPS connection on localhost. That's
+  why we try and make this work as efficiently, and securely as possible.
+
+  We generate a unique certificate for each Bankai installation. This means
```

Comment:
Could add filepath to generated Bankai cert files here?

"We generate a unique certificate for each Bankai installation at `~/.config/bankai/`."

Reply:
done!